Add registry tip to ocb (#5028)
open-telemetry/opentelemetry.io: content/en/docs/collector/custom-collector.md

      go.opentelemetry.io/collector/receiver/otlpreceiver {{% version-from-registry collector-receiver-otlp %}}
```

{{% alert color="primary" title="Tip" %}}

> To find components that you can add to your custom Collector, take a look the

Reviewer: ```suggestion
To find components that you can add to your custom Collector, take a look at the
```

      go.opentelemetry.io/collector/receiver/otlpreceiver {{% version-from-registry collector-receiver-otlp %}}
```

{{% alert color="primary" title="Tip" %}}

> To find components that you can add to your collector component, take a look

Reviewer 2: ```suggestion
To find components that you can add to your Collector component, browse
```

To your Collector component or Collector components?

Author: custom Collector probably better?

Author: ```suggestion
To find components that you can add to your custom Collector, take a look
```

> into the
> [OpenTelemetry Registry](https://opentelemetry.io/ecosystem/registry/?language=collector).
> You will not only find what is available, but also the full string you need to
> add to your `builder-config.yaml`.

Reviewer 2: ```suggestion
the
[OpenTelemetry Registry](https://opentelemetry.io/ecosystem/registry/?language=collector).
In addition to available components, the Registry provides the full string you need to
add to your `builder-config.yaml`.
```

> To find components that you can add to your custom Collector, take a look
> the
> [OpenTelemetry Registry](https://opentelemetry.io/ecosystem/registry/?language=collector).
> In addition to available components, the Registry provides the full string you need to
> add to your `builder-config.yaml`.

Reviewer 3: ```suggestion
For a list of components that you can add to your custom collector, see the [OpenTelemetry Registry](/ecosystem/registry/?language=collector). Note that registry entries provide the full string you need to add to your `builder-config.yaml`.
```

By the way, what is "the full string" -- i.e., it is a full string of what? The component name or ID?

Reviewer 3: Btw, IMHO, "collector" should be in lowercase (as I suggest in the previous comment) in "custom collector" because we're not talking about _the_ OTel Collector. WDYT?

Author: the "full string" is something like this:

```
exporters:
    - gomod:
        github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter v0.108.0
```

I see how this is not clear, the "full component name and version you need" might be better?

It's still an OpenTelemetry Collector, it's just a custom one, I guess?

Reviewer 3: > I see how this is not clear, the "full component name and version you need" might be better?

Yes. I see that you've made this change already. Thanks.

> It's still an OpenTelemetry Collector, it's just a custom one, I guess?

Well, the fact that you write "_an_ OpenTelemetry Collector" means that even you don't think that it is _the_ Collector. IMHO, _the_ Collector should refer to a fixed (for any given version) binary. Does that make sense?

Author: Hm, there is no fixed binary of the collector afaik, even for the ones we deliver as community there is "Core" and "Contrib" and some other distributions. My understanding so far was, that we say Collector when we mean an OpenTelemetry collector build from https://github.com/open-telemetry/opentelemetry-collector, this includes the "Whatever Distribution of OpenTelemetry Collector", and custom Collectors, but if someone has a component that collects telemetry or whatever it's a collector.

Author: @chalin when you are back, please take a look

      go.opentelemetry.io/collector/receiver/otlpreceiver {{% version-from-registry collector-receiver-otlp %}}
```

{{% alert color="primary" title="Tip" %}}

> To find components that you can add to your custom Collector, take a look at the
> [OpenTelemetry Registry](https://opentelemetry.io/ecosystem/registry/?language=collector).
> In addition to available components, the Registry provides the full name and version you
> need to add to your `builder-config.yaml`.

Reviewer 3: Reiterating copyedits from before that seem to have gotten lost:

```suggestion
For a list of components that you can add to your custom collector, see the [OpenTelemetry Registry](/ecosystem/registry/?language=collector). Note that registry entries provide the full name and version you need to add to your `builder-config.yaml`.
```
